src/mam4xx/lin_strat_chem.hpp

```diff
@@ -259,7 +259,7 @@ void lin_strat_chem_solve(
 } // lin_strat_chem_solve
 
 KOKKOS_INLINE_FUNCTION
-void lin_strat_sfcsink_kk(const Real delta_t, const Real pdel, // in
+Real lin_strat_sfcsink_kk(const Real delta_t, const Real pdel, // in
                           Real &o3l_vmr, const Real o3_sfc, const Real o3_tau,
```

Comment:
would you need to remove `o3l_vmr` from the function signature?

Reply:
It is both input and output, so we need to keep both.